src/demos/pick-list/basic.html

```diff
@@ -7,17 +7,48 @@
     <script src="../_assets/head.js"></script>
 
     <style>
+      .parent {
+        display: flex;
+        align-items: center;
+        margin: 25px 0;
+      }
+
+      .child {
+        flex: 0 0 40%;
+        margin: 0 25px;
+        color: var(--calcite-ui-text-3);
+        font-family: var(--calcite-sans-family);
+        font-size: var(--calcite-font-size-0);
+        font-weight: var(--calcite-font-weight-medium);
+        padding: 10px 0;
+      }
+
+      .right-aligned-text {
+        text-align: right;
+        flex: 0 0 15%;
+      }
+
+      hr {
+        margin: 35px 0;
+        border-top: 1px solid var(--calcite-ui-border-2);
+      }
+
       calcite-pick-list {
         margin-bottom: 1rem;
       }
     </style>
   </head>
   <body>
-    <main>
-      <section class="example-container">
-        <h1>calcite-pick-list</h1>
+    <div>
+      <calcite-button href="/" icon-start="arrow-left" color="neutral"> Home </calcite-button>
```

Comment:
Hey @ManahariDahal can you merge in 4master and run npm install? This should run prettier and format the HTML files here the next time before a commit happens. I'm noticing ` Home ` shouldn't have spaces around it.

Since the automatic prettier formatting won't happen until next commit. Can you run `npm run prettier` and check in the files it changes that pertain to this PR?

Reply:
do I need to run `npm install` every time I am in a new branch?

Reply:
No, just to get the `husky install` to run once

Reply:
weird, prettier worked on every line apart from the `Home`